CONTRIBUTING.md

```diff
@@ -99,6 +98,26 @@ Install pre commit hooks:
 pre-commit install
 ```
 
+Set environment variables for your setup. You can use [direnv](https://direnv.net/) to do this automatically.
```

Comment:
Maybe we can drop the env from here that has the same pre-defined value? Like `DEFAULT_CHANNEL_SLUG` (it already has `default-channel` value by default)
Also, instead of setting up some envs without explanation, I would suggest providing a link with an explanation for each of the envs that you need to set up. Most of the envs are described here: https://docs.saleor.io/setup/configuration#general-configuration

Reply:
I'll put better docs in poe setup, since it supports loading from env and you don't need `direnv` for that